### PR TITLE
SITL: unhide groups when generating parameters in autotest

### PR DESCRIPTION
**Pull request by peterbarker**

### CODE REVIEW

File: libraries/AP_Param/AP_Param.cpp [hunk 1]
```diff
@@ -78,6 +78,8 @@ void AP_Param::eeprom_write_check(const void *ptr, uint16_t ofs, uint8_t size)
     _storage.write_block(ofs, ptr, size);
 }
 
+bool AP_Param::_hide_disabled_groups = false;
```
Comment (OXINARF) [inline libraries/AP_Param/AP_Param.cpp]: Is this correct? By default we don't hide disabled groups?

Comment (peterbarker) [inline libraries/AP_Param/AP_Param.cpp]: Darn.  No idea how that happened.  I'll blame "too late at night".

**Comment by peterbarker** [comment]
@OXINARF Commit now split.

**Comment by tridge** [comment]
looks great, thanks! just waiting for travis

**Comment by OXINARF** [comment]
@tridge Finished! Merge please :smiley:

**Pull request merged**